package.json

```diff
@@ -27,7 +27,7 @@
     "spellcheck": "npm run spellcheck:ts && npm run spellcheck:md",
     "spellcheck:ts": "npx cspell --gitignore -e \"./packages/ethereum-tests\" -e \"./packages/wallet/test\" -e \"./packages/client/archive\" -c ./config/cspell-ts.json \"./packages/**/*.ts\" --cache --show-suggestions --show-context",
     "spellcheck:md": "npx cspell --gitignore -e \"./packages/ethereum-tests\" -e \"./packages/client/withdrawals-testnet/**\" -e \"./packages/**/docs\" -c ./config/cspell-md.json \"**.md\" --cache --show-suggestions --show-context",
-    "install-browser-deps": "npm install webdriverio@9.8.0 @vitest/browser@3.0.8"
+    "install-browser-deps": "npm install webdriverio@9.8.0 playwright@1.15.1 @vitest/browser@3.0.8"
```

Comment:
Do we still need webdriver if we are switching to playwright?

Reply:
Have removed